kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -45,6 +46,9 @@ public class FunSpec private constructor(
   public val modifiers: Set<KModifier> = builder.modifiers.toImmutableSet()
   public val typeVariables: List<TypeVariableName> = builder.typeVariables.toImmutableList()
   public val receiverType: TypeName? = builder.receiverType
+
+  @DelicateKotlinPoetApi("Context receivers are currently a preview feature")
```

Comment:
Hmmm, this isn't the right one either. I thought we had an existing experimental annotation API (`@ExperimentalKotlinPoetApi` or similar). If there isn't, please make one and use that

Reply:
I thought it wasn't the right one, but even though there are Experimental/Preview annotations these are for other modules like the ksp interop